devoir/pavage.ipynb

```diff
@@ -0,0 +1,313 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.            self.casesWeight=[0 for i in self.cases]
</pre><p>casesWeight n'a l'air pas utilisé ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fortierq/oc-m1-2021/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>